libr/util/print.c

```diff
@@ -2572,6 +2572,37 @@ R_API void r_print_hex_from_bin(RPrint *p, char *bin_str) {
 	free (buf);
 }
 
+R_API void r_print_bin_from_str(RPrint *p, char *str) {
+	int i = 0;
+	int len = strlen (str);
+	for (i = 0; i < len; i++) {
```

Comment:
this is the same as calling the r_str_str2binstr, dont duplicate code. the only difference is the PrintfCallback function pointer which can be assigned before the loop